Automattic/newspack-theme
# feat: add post.news social icon

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds an icon for https://post.news to the Social Links menu.

Closes `1203451787793554/1203492348261144`

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Edit your social links menu and add a link that starts with `https://post.news` (example: https://post.news/roughdraft) and publish.
3. Confirm that the link uses the Post News icon, and not the placeholder (the icon looks like a comment bubble with a P. inside). 
4. Wherever your menu is appearing (header or footer), swap your background colour to make sure the new icon picks up the correct colour (if it's on a dark background it should be white; if it's on a white background, it should be dark). 

![image](https://user-images.githubusercontent.com/177561/207943092-ab5af4ad-9773-4ed4-b8c8-7b4420c3d9cc.png)

### Other information:
